templates/identity/email-password/register-partial.html.ejs

```diff
@@ -107,13 +107,6 @@
       </li>
     </ol>
   </div>
-  <div id="noPasswordForm" class="hidden">
```

Comment:
Could you just explain this one? Forgot why it's no longer needed.

Reply:
Point 3 in the first comment way above:
> Due to the account changes it is now always required to provide a password when creating a pod, even if you're not registering with the IDP. There are ways around it, but at some point in the future we probably want to provide some extra pod options which could be accessed using the same login details.